content/components/link.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Link
+description: Links are used to apply styles to hyperlink text.
+reactId: link
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=20953%3A78940&t=wIvqJYawyRBUeGPt-1
+---
+
+import {Box, Text} from '@primer/react'
+import Code from '@primer/gatsby-theme-doctocat/src/components/code'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+Links enable users to navigate between pages or resources in a seamless way. Links can be used within a body of text or as a standalone element.
+
+Links navigate you to a new place or new content. **Do not** use links for triggering an action. Instead, use [buttons](/components/button), which are designed to activate a feature.
+
+Links should be concise and not exceed more than a few words.
+
+## Anatomy
+
+The link component can be displayed in default, muted, or blue colors. Links can be underlined, may include a leading visual or trailing visual, and can be either the default or small size. 
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 3}}
+>
+  <img
+    width="400"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/586552/225489084-4f4357b4-4c47-4a55-8770-76e03b8c500f.png"
+    
+  />
+  <img
+    width="369"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/586552/225489086-e604fbc8-e449-416e-bd26-cc80408c5d8d.png"
+   />
+</Box>
+
+## Accessibility
+
+Links play a key part in your experience on the web, but without proper consideration they can be frustrating to use, skipped over, or completely unnoticed.
+
+For screen reader assistive technology, links and buttons are expected to function differently from each other. If a link is activated and does not do what was expected, that can be disorienting and frustrating.
```

Comment:
```suggestion
For assistive technologies like screen readers, links and buttons are expected to function differently from each other. If a link is activated and does not do what was expected, that can be disorienting and frustrating.
```

I think this may affect more than just screen readers, so I rephrased this to be slightly more inclusive of other ATs.